ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -60,19 +63,34 @@ def cmd(self, cmds):
 
         return stdout, stderr, return_code
 
-    def ssh(self, cmds):
-        ssh_cmds = ["ssh", "-oStrictHostKeyChecking=no", "-oServerAliveInterval=2", "admin@" + self.dut_ssh]
-        ssh_cmds.extend(cmds)
-        stdout, stderr, return_code = self.cmd(ssh_cmds)
-        if stdout != []:
-            self.log("stdout from dut: '%s'" % str(stdout))
-        if stderr != []:
-            self.log("stderr from dut '%s'" % str(stderr))
-        self.log("return code from dut: '%s'" % str(return_code))
+    def dut_exec_cmd(self, cmd):
+        client = paramiko.SSHClient()
+        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+        stdout = stderr = []
+        return_code = 1
+
+        try:
+            client.connect(self.dut_ssh, username=self.dut_username, password=self.dut_password, allow_agent=False)
+            si, so, se = client.exec_command(cmd, timeout=30)
+            stdout = so.readlines()
+            stderr = se.readlines()
+            self.log('executed command {}, stdout={}, stderr={}'.format(cmd, stdout, stderr))
+            return_code = 0
```

Comment:
Can we get return code of exec_command? Is it possible to ignore some error if we always return 0 here?

Reply:
The paramiko library uses pattern of sending cmd string and receiving returned results in streams. There is no return code returned by the exec_command method of the paramiko.SSHClient.

Reply:
can we reuse this [class](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/ptftests/device_connection.py)?

Reply:
Sure, pushed a new commit.